### PR TITLE
treehouses commands on rpi only (fixes #245)

### LINKED ISSUE #245: don't show commands that can't be run outside the pi

### PR DESCRIPTION
**Pull request by hiroTochigi**

### CODE REVIEW

**Comment by dogi** [comment]
@hiroTochigi think that that last change is good

File: cli.sh [hunk 1]
```diff
@@ -59,10 +60,12 @@ case $1 in
     rename "$2"
     ;;
   password)
+    checkrpi
     checkroot
     password "$2"
     ;;
   sshkey)
+    checkrpi
```
Comment (dogi) [inline cli.sh]: we wanna change only on root when nonrpi system,
but not "/home/pi/.ssh/authorized_keys"

File: modules/sshkey.sh [hunk 2]
```diff
@@ -21,18 +23,28 @@ function sshkey () {
     cat /root/.ssh/authorized_keys
 
     echo "==== pi keys ===="
```
Comment (irisb1701) [inline modules/sshkey.sh]: @hiroTochigi This line should go inside the if statement

File: modules/sshkey.sh [hunk 3]
```diff
@@ -21,18 +23,28 @@ function sshkey () {
...
+      cat /home/pi/.ssh/authorized_keys
+    fi
+
   elif [ "$1" == "delete" ]; then
     if [ -z "$2" ]; then
       echo "Error: missing argument"
       echo "Usage: $(basename "$0") sshkey delete \"<key>\""
       exit 1
     fi
     sed -i "/^$2/d" /home/pi/.ssh/authorized_keys
```
Comment (dogi) [inline modules/sshkey.sh]: @hiroTochigi this is the one to be in the if then not the following one

File: modules/sshkey.sh [hunk 4]
```diff
@@ -21,18 +23,28 @@ function sshkey () {
...
-    sed -i "/^$2/d" /root/.ssh/authorized_keys
+
+    if [ "$(detectrpi)" != "nonrpi" ]; then
+      sed -i "/^$2/d" /root/.ssh/authorized_keys
+    fi
+
   elif [ "$1" == "deleteall" ]; then
     rm /home/pi/.ssh/authorized_keys
```
Comment (dogi) [inline modules/sshkey.sh]: @hiroTochigi this is the one to be in the if then not the following one

File: modules/sshkey.sh [hunk 5]
```diff
@@ -7,10 +7,12 @@ function sshkey () {
...
-    chown -R pi:pi /home/pi/.ssh
-
+    if [ "$(detectrpi)" != "nonrpi" ]; then
+      echo "$@" >> /home/pi/.ssh/authorized_keys
+      chmod 600 /home/pi/.ssh/authorized_keys
+      chown -R pi:pi /home/pi/.ssh
+    fi
+    
     echo "$@" >> /root/.ssh/authorized_keys
     chmod 600 /root/.ssh/authorized_keys
```
Comment (dogi) [inline modules/sshkey.sh]: echo line underneath needs also to say the right thing

File: cli.sh [hunk 6]
```diff
@@ -92,14 +96,17 @@ case $1 in
     container "$2"
```
Comment (dogi) [inline cli.sh]: think we nned a checkrpi also here

**Pull request merged**